### ISSUE TITLE
Invalid prop type supplied to ListControl 

### ISSUE THREAD

**Issue by drlogout** [netlify/netlify-cms#822]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
If a date or a datetime widget is put inside a list widget it shows a warning in the console:

`Failed prop type: Invalid prop "value" supplied to "ListControl", expected a ReactNode.`

**- If the current behavior is a bug, please provide the steps to reproduce.**
Change the example config.yml to:

```yml
  - name: "dates"
    label: "Dates"
    folder: "_dates"
    slug: "{{slug}}"
    create: true
    fields:
      - {label: "Title", name: "title", widget: "string", tagname: "h1"}
      - name: events
        label: Events
        widget: list
        fields:
          - {label: "Title", name: "title", widget: "string"}
          - {label: "Start Date", name: "start_date", widget: "date"}
```

**- What is the expected behavior?**
There should be no warning.

**- Please mention your CMS, node.js, and operating system version.**
Netlify CMS: 0.7.4-dev
Node: v8.6.0

**Comment by erquhart** [comment]
I've noticed this too, not sure why we have the list widget expecting a react node as a value - it will always be an array, and almost always an array of objects.

**Comment by erquhart** [comment]
Fixed in 1.0.
